spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -667,7 +683,7 @@ function revertInFlightChanges(sentPacket: Packet, receivedPacket: Packet) {
   // the token on our chain is the token in the sentPacket
   for token in sentPacket.tokens {
     // check if the packet we received was a source token for our chain
-    if isSource(receivedPacket.destinationPort, receivedPacket.desinationChannel, token) {
+    if isTracePrefixed(receivedPacket.destinationPort, receivedPacket.desinationChannel, token) {
```

Comment:
```suggestion
    // we are checking if the tokens that were sent out by our chain in the sentPacket were source tokens
    // with respect to the original receivedPacket. If the tokens on our chain were prefixed by our channelEnd's
    // identifiers, then it was a minted voucher and we need to burn it.
    // Otherwise, it was an original token from our chain and we must give the tokens back to the escrow account
    if !isTracePrefixed(receivedPacket.destinationPort, receivedPacket.desinationChannel, token) {
```

Reply:
I believe the code as you have it is wrong.

If the token is prefixed by the receivedPacket destPort and destChannel, that would imply that we minted it during the receive and thus it is **not** a source token, right?

Reply:
You're right, with the change from `isSource` to `isTracePrefixed` the condition became wrong. Thank you for reviewing and for the attention to detail!